patches/4541080F - The Orange Box (engine_360.dll).patch.toml

```diff
@@ -23,6 +23,21 @@ hash = "B433D6E33726406B" # engine_360.dll
     [[patch.be32]]
         address = 0x8614ab0c
         value = 0x39600001 # li r11,0x1
+    [[patch.be16]]
+        address = 0x8625480a
+        value = 0x3630
```

Comment:
What does it do?

Reply:
Changes framerate float from 30 to 1000:
![image](https://user-images.githubusercontent.com/3462541/167464577-4f88829f-b350-4097-8953-1a3812d57509.png)